CRM/Core/PrevNextCache/Interface.php

```diff
@@ -89,20 +87,24 @@ public function getSelection($cacheKey, $action = 'get');
    *
    * @param string $cacheKey
    * @param int $id1
-   * @param int $id2
    *
    * @return array
+   *   List of neighbors.
+   *   [
+   *     'foundEntry' => 1,
+   *     'prev' => ['id1' => 123, 'data'=>'foo'],
+   *     'next' => ['id1' => 456, 'data'=>'foo'],
+   *   ]
    */
-  public function getPositions($cacheKey, $id1, $id2);
+  public function getPositions($cacheKey, $id1);
```

Comment:
My grep shows that there are places this is called which are not removed in this pr - e.g CRM_Contact_Page_View

```
      $pos = Civi::service('prevnext')->getPositions("civicrm search $qfKey",
        $this->_contactId,
        $this->_contactId
      );
```

I note this doesn't have to be blocking as it won't cause breakage & it might be covered in your subsequent PR